tas-clients/resources/consoleclidownload-cosign.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleCLIDownload
+metadata:
+  name: cosign
+spec:
+  description: cosign is a CLI tool that allows you to
+    manage sigstore artifacts.
+  displayName: cosign - Command Line Interface (CLI)
+  links:
+  - href: https://tas-clients-trusted-artifact-signer.apps.open-svc-sts.k1wl.p1.openshiftapps.com/clients/cosign
```

Comment:
This gets replaced with the `OPENSHIFT_BASE_DOMAIN` value?

Reply:
Yup - this will be dynamic with the helm template files & values.yaml substitution - I just have to move it over to the templates folder and add some {{ blah.blah.blah }} - and also this will be a new section in our chart.

Reply:
updated!